internal/provisioners/default/zz-default.provisioners.yaml

```diff
@@ -618,3 +618,130 @@
         ports:
         - port: 27017
           targetPort: 27017
+
+
+- uri: template://default-provisioners/rabbitmq
+  type: amqp
+  init: |
+    randomVHost: vhost-{{ randAlpha 8 }}
+    randomUsername: user-{{ randAlpha 8 }}
+    randomPassword: {{ randAlphaNum 16 | quote }}
+
+  state: |
+    service: rabbitmq-{{ .SourceWorkload }}-{{ substr 0 8 .Guid | lower }}
+    vhost: {{ dig "vhost" .Init.randomVHost .State | quote }}
+    username: {{ dig "username" .Init.randomUsername .State | quote }}
+    password: {{ dig "password" .Init.randomPassword .State | quote }}
+
+  outputs: |
+    host: {{ .State.service }}
```

Comment:
Please align the outputs here with the ones in compose: https://github.com/score-spec/score-compose/blob/293dfaf59852bc19a93bae16d3be41c6d9d55d55/internal/command/default.provisioners.yaml#L302 and the documented outputs `host, port, vhost, username, password`

remove management_port, connection, and make the port an integer :)

The reason we remove management_port from the outputs is that not all amqp implementations have a management port like this, and the resource passed to the workload is only concerned with the pure amqp connection, not with any back-office ui.

The rest of it looks good :)